core/tee/tee_svc_storage.c

```diff
@@ -25,19 +25,44 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <tee/tee_svc_storage.h>
-
 #include <kernel/tee_ta_manager.h>
-#include <tee_api_defines.h>
 #include <kernel/tee_misc.h>
+#include <mm/tee_mmu.h>
 #include <tee/tee_fs.h>
 #include <tee/tee_fs_defs.h>
 #include <tee/tee_obj.h>
 #include <tee/tee_svc.h>
-#include <mm/tee_mmu.h>
 #include <tee/tee_pobj.h>
+#include <tee/tee_svc_storage.h>
+#include <tee_api_defines.h>
+#include <tee_api_defines_extensions.h>
 #include <trace.h>
 
+static const struct tee_file_operations *file_ops(uint32_t storage_id)
+{
+
+	switch (storage_id) {
+	case TEE_STORAGE_PRIVATE:
+#if defined(CFG_REE_FS)
```

Comment:
I would add the comment you gave in the commit message, that is `The value TEE_STORAGE_PRIVATE will select the REE FS if available, otherwise RPMB`